docs/user-guide/about-rules.md

```diff
@@ -102,7 +102,7 @@ a {}
 
 Additionally, some whitespace rule make use of another set of keywords:
 
--   `comma`, `colon`, `semicolon`, `opening-brace`, `closing-brace`, `opening-parenthesis`, `closing-parenthesis`, `operator` or `range-operator` are used if a specific piece of punctuation in the *thing* is being targetted
+-   `comma`, `colon`, `semicolon`, `opening-brace`, `closing-brace`, `opening-parenthesis`, `closing-parenthesis`, `operator` or `range-operator` are used if a specific piece of punctuation in the *thing* is being targeted
```

Comment:
Please, note that [targetted](https://en.wiktionary.org/wiki/targetted) is UK version, while [targeted](https://en.wiktionary.org/wiki/targeted) is US version. Not sure which variation of English is used in the docs...